muxplore/main.c

```diff
@@ -248,58 +248,37 @@ char *load_content_description() {
         }
             break;
         default: {
-            char *card_full;
-            switch (module) {
-                case MMC:
-                    card_full = SD1;
-                    break;
-                case SDCARD:
-                    card_full = SD2;
-                    break;
-                case USB:
-                    card_full = E_USB;
-                    break;
-                default:
-                    return NULL;
-                    break;
-            }
+            char *desc_name = strip_ext(items[current_item_index].name);
 
-            if (strcasecmp(get_last_subdir(sd_dir, '/', 4), strip_dir(card_full)) == 0) {
```

Comment:
This check is not needed here as the check for Root happens above and will never hit this spot when current location is root